<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtuDevice</class>
 <widget class="QFrame" name="AtuDevice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>176</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ATU</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="btnConnect">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btnDisconnect">
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblErrorCode">
       <property name="text">
        <string>Error code</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="sbErrorCode">
       <property name="maximum">
        <number>9</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="lblState">
     <property name="text">
      <string>&lt;STATE&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="lblTunedFrequency">
     <property name="text">
      <string>&lt;TUNED FREQUENCY&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
